pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java

```diff
@@ -2382,6 +2382,9 @@ protected void internalSetSchemaValidationEnforced(boolean schemaValidationEnfor
     protected boolean internalGetIsAllowAutoUpdateSchema() {
         validateNamespacePolicyOperation(namespaceName, PolicyName.SCHEMA_COMPATIBILITY_STRATEGY,
                 PolicyOperation.READ);
+        if (getNamespacePolicies(namespaceName).is_allow_auto_update_schema == null) {
+            return pulsar().getConfig().isAllowAutoUpdateSchemaEnabled();
+        }
```

Comment:
Hmmm, yes. No good choice here. In principle, we should not return the broker-level configuration. I think we can correct the API in v3 REST API to make sure returns null if the namespace level doesn't have the `AllowAutoUpdateSchema`.

Reply:
Maybe we could add the param `applied` for getting policies methods, such as topic policies. If the param `applied` is true, the method returns an applied value or returns a setting value in namespace level(if not set, the value is null).

Reply:
+1.